bin/eslint.js

```diff
@@ -7,7 +7,12 @@ var exitCode = 0,
 
 if (useStdIn) {
     process.stdin.pipe(concat({ encoding: "string" }, function(text) {
-        exitCode = cli.execute(process.argv, text);
+        try {
+            exitCode = cli.execute(process.argv, text);
+        } catch (ex) {
+            console.error(ex.message);
```

Comment:
Is there any reason you don’t want to show the stacktrace in the console? This would actually lead to two different outputs for the same error. If `cli.execute` throws on line 18, it would output the stacktrace, if it throws on line 11 it only outputs the message.